src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -43,8 +43,12 @@ int_ipv6auto_tooltip : T('Set to automatically configure the IPv6 address with\
  target="_blank">rtsol(8)</a>. Only one interface can\
  be configured this way.'),
 
-disable_offload_capabilities_placeholder: T('Disable offload capabilities'),
-disable_offload_capabilities_tooltip: T('Set to disable NIC capabilities when using a Jail or VM.'),
+disable_offload_capabilities_placeholder: T('Disable Hardware Offloading'),
```

Comment:
If the logic was flipped, this can just be "Use Hardware Offloading".

Reply:
Ticket to reverse the logic: https://jira.ixsystems.com/browse/NAS-104388